dali/operators/audio/resample.h

```diff
@@ -125,7 +128,8 @@ class ResampleBase : public Operator<Backend> {
           DALI_FAIL(make_string("Cannot produce a non-empty signal from an empty input.\n"
             "Error at sample ", s));
         }
-        scales_[s] = in_length ? 1.0 * out_length / in_length : 0.0;
+        args_[s].in_rate = 1.0;
+        args_[s].out_rate = in_length ? 1.0 * out_length / in_length : 0.0;
```

Comment:
Now that we have `args`, this would increase precision:
```suggestion
        args_[s].in_rate = in_length ? in_length : 1;    // avoid division by 0
        args_[s].out_rate = out_length ? out_length : 1;  // avoid division by 0
```